### ISSUE TITLE
Automatic maximising control

### ISSUE THREAD

**Issue by ocots** [control-toolbox/indirect#1]
We could compute the control by its value at initial time and thanks to IFT but also by a DAE solver inside the shooting function. Besides, if we solve the BVP, we can do the same.

**Comment by jbcaillau** [comment]
@ocots yes. example available on solar sail application, check section 3.3 of [this preprint](http://caillau.perso.math.cnrs.fr/research/ocam-2023.pdf)
